qiskit/pulse/timeslots.py

```diff
@@ -98,6 +98,13 @@ def __eq__(self, other):
             return True
         return False
 
+    def __repr__(self):
+        """Return a readable representation of Interval Object"""
+        start_time = self._begin
+        end_time = self._end
+        rep = self.__class__.__name__ + "({}, {})".format(start_time, end_time)
+        return rep
```

Comment:
I think this would be quite readable as a one liner. What do you think of this:
```
return self.__class__.__name__ + "({}, {})".format(self.begin, self.end)
```
Also, better to use `.begin` than `._begin`

Reply:
Yeah putting that as one line would probably be better considering it still describes what is be put into the string. 

I will also make the changes to calling the attributes.

Reply:
Should I change 
```
    def __repr__(self):
        interval = self._interval
        channel = self._channel
        rep = self.__class__.__name__ + "({}, {})".format(channel, (interval.begin, interval.end))
        return rep
```
to 
```
    def __repr__(self):
        return self.__class__.__name__ + "({}, {})".format(self.channel,
                                                           (self.interval.begin, self.interval.end))
```
as well?

Reply:
yeah, while we're at it, can we keep the formatting consistent? e.g.

```
return "{class}({channel}, {interval})".format(self.__class__.__name__,
                                               self.channel,
                                               (self.interval.begin, self.interval.end))
```
You can use this style for `Interval`, too

Reply:
That last comment is probably more of a personal preference, though ^